pkg/legacy/tarball/write.go

```diff
@@ -297,10 +298,15 @@ func MultiWrite(refToImage map[name.Reference]v1.Image, w io.Writer) error {
 		addTags(repos, tags, prev.config.ID)
 	}
 
+	sort.Slice(m, func(i, j int) bool {
```

Comment:
Do we not care about the order in which these images are written to the tarball? Just the manifest.json file that describes them?

Reply:
Not sure. I have tried in my use-case and this seems to work. I will have a look and validate that is the case.

Reply:
You were correct, I sorted all images by their `ConfigName()` so now all tarball contents are also sorted.